src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+body {   font-family: 'Roboto', sans-serif;
   margin: 0;
+  background-color: aliceblue;
+  font-size: 12px;
+  text-transform: uppercase;
 }
+.header{
+  background-color: #fff;
+width:  100%;
+height: 60px ;
+display: flex;
+justify-content: space-between;
+
+}
+
+.image{
+  height: 40px;
+  width: 40px;
+  background-color: #fff;
+margin-left: 50px;
+}
+
+.nav{
+  display: inline;
```

Comment:
The `.nav` class is set to `display: inline;`, which might not be the best choice for a navigation bar. Consider using `display: flex;` or `display: block;` to better control the layout of navigation items.